FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %}
+
+{% block heading %}
+    Students
+{% endblock %}
+
+{% block content %}
+<head>
+<style>
+table {
+    border-collapse: collapse;
+    width: 50%;
+}
+
+td, th {
+    border: 1px solid #dddddd;
+    text-align: left;
+    padding: 8px;
+}
+
+tr:nth-child(even) {
+    background-color: #dddddd;
+}
+
+</style>
+</head>
+<body>
+
+
+{% if student_list %}
+    <div>
+    <table align = "center">
+     <tr>
+      <th>CCID</th>
+      <th>Year of Study</th>
+      <th>Program</th>
+     </tr>
+    {% for student in student_list %}
+        <tr>
+         <td>{{ student.ccid }}</td>
+         <td>{{ student.year }}</td>
+         <td>{{ student.program }}</td>
+         <td><a style="color:blue;" href="/student/{{ student.ccid }}/">View</a></td>
```

Comment:
Style should be left in the css. If we need this link to appear differently from the exiting banner links, consider adding a class to this anchor tag (or even better, add a class to the banner anchors since they are the exception) so we can differentiate them. 

Also, the text inside this anchor needs to be translatable too.